posts/TechYesWeCan_Pt1.md

```diff
@@ -0,0 +1,115 @@
+---
+authors:
+- Leigha Dellinger, Sarah Narum, Rita Barlow, Trish Brumett, Erin Geoghan, Ramya Shetty
+tags:
+- Women In Technology
+- RVATech
+date: 2018-10-31T12:21:50.000Z
+title: "Tech Yes We Can! Version 1.1"
+image: 
+---
+
+# 'Tech Yes She Can!’ Version 1.1
+
+On October 18, 2018, six of IpponRVA’s consultants attended the RVATech Women in Technology conference here in Richmond Virginia. As advertised, “Central Virginia’s premier technology conference brings together professionals on a wide range of technology topics to learn, network, and collaborate.” After attending the conference and discussing the day at the conference happy hour we decided that we should share our opinions of the day with the world, or whomever would listen! 
+
+It was advised, and properly so, that the ramblings of six women and their experience at a conference may be a bit overwhelming to society, so we broke this inspirational knowledge fest into a two-parter. We hope that you find something here to inspire you, regardless of gender, to go out into the world and make a difference, even if that difference is solely introspective. We have done our best to carry on the essence of the conference within our tales and more importantly bring along our own experiences in order to offer some sort of guidance if you are here to seek it. Although we do not speak for RVATech or all women, we do speak as six women IN technology from various backgrounds, points in our careers, and stylistic preferences. Take what you want, leave what you don’t and comments or feedback are encouraged!
+
+
+>## Sarah Narum
+>Ippon Engineer | Data | snarum@ipponusa.com
+
+This was my second year attending the RVATech: Women in Technology conference and once again I was blown away by the vivacious, empowering women who led (and attended) this conference. The ever-so-energetic MC, Courtney Ferrell, summarized it best when she said, ‘you need three things to be successful: Confidence, Creative Spirit and Connections’. All of the speakers touched on the importance of these three factors of success in their own way so I’ve decided to address each:
```

Comment:
Use double quotes to quote text throughout the post

Reply:
I copied and pasted from three authors so I'm hoping I caught them all!